packages/cli/src/oclif/commands/invoke.js

```diff
@@ -259,7 +259,7 @@ const getAuthLabel = async (labelTemplate, authData, meta, zcacheTestObj) => {
   const testResult = await testAuth(authData, meta, zcacheTestObj);
   labelTemplate = labelTemplate.replace('__', '.');
   const tpl = _.template(labelTemplate, { interpolate: /{{([\s\S]+?)}}/g });
-  return tpl(testResult);
+  return tpl({ bundle: { authData, inputData: testResult } });
```

Comment:
out of curiosity, how come the testResult is added into inputData as oppose to outputData?

Reply:
I'm about 80% sure. It's because 1) the test result is input for `connectionLabel` and 2) `outputData` was only introduced later for other method calls.